src/test/java/com/ea/restaurant/service/impl/utils/fixtures/UserFixture.java

```diff
@@ -0,0 +1,73 @@
+package com.ea.restaurant.service.impl.utils.fixtures;
+
+import com.ea.restaurant.constants.Roles;
+import com.ea.restaurant.constants.Status;
+import com.ea.restaurant.constants.UserType;
+import com.ea.restaurant.entities.User;
+import java.time.Instant;
+
+public final class UserFixture {
+
+  public static final Long id = 1L;
+  public static final String name = "juan";
+  public static final String lastName = "perez";
+  public static final String username = "jp001";
+  public static final String password = "1234";
+  public static final Roles role = Roles.ADMINISTRATOR;
+  public static final UserType type = UserType.INTERNAL;
+  public static final Long createdBy = 1L;
+  public static final Instant createdDate = Instant.now();
+  public static final Status entityStatus = Status.ACTIVE;
+
+  public static User buildUser() {
```

Comment:
Why are you repeating this function every time? shouldn't you just re use existing functions?

Reply:
Done.